src/numato_gpio/__init__.py

```diff
@@ -436,7 +441,7 @@ def _write(self, query):
 
     def _read_string(self, expected):
         string = self._read(len(expected.encode()))
-        if string != expected:
+        if string.lower() != expected:
```

Comment:
Please add a comment from which device type you expect a response in capital letters.